keyboards/bolsa/bolsalice/info.json

```diff
@@ -0,0 +1,81 @@
+{
+    "keyboard_name": "Bolsalice",
+    "manufacturer": "BolsaSupply",
+    "usb": {
+        "pid": "0x414C",
+        "vid": "0x4253",
+        "device_version": "0.0.1"
+    },
+    "url": "",
+    "maintainer": "qmk",
+    "layouts": {
+        "LAYOUT": {
```

Comment:
The layouts you have in here and the keyboard.h do not make sense.
![image](https://user-images.githubusercontent.com/4781841/197096017-e1c3ecc1-2ea8-46a4-b599-884068df3db5.png)

Reply:
They've been fixed now